### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: Admin/GalleryItemAdmin.php [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\Admin;
-
-use Sonata\AdminBundle\Admin\AbstractAdmin;
-use Sonata\AdminBundle\Datagrid\ListMapper;
-use Sonata\AdminBundle\Form\FormMapper;
-
-class GalleryItemAdmin extends AbstractAdmin
```
Comment (jordisala1991) [inline Admin/GalleryItemAdmin.php]: Careful, this class should not be removed

File: Document/BaseGalleryItem.php [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\Document;
-
-use Sonata\MediaBundle\Model\GalleryItem;
-
-abstract class BaseGalleryItem extends GalleryItem
```
Comment (jordisala1991) [inline Document/BaseGalleryItem.php]: This class should not get removed

File: Entity/BaseGalleryItem.php [hunk 3]
```diff
@@ -1,28 +0,0 @@
...
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\Entity;
-
-use Sonata\MediaBundle\Model\GalleryItem;
-
-abstract class BaseGalleryItem extends GalleryItem
```
Comment (jordisala1991) [inline Entity/BaseGalleryItem.php]: Same for this class

File: Form/Type/ApiGalleryItemType.php [hunk 4]
```diff
@@ -1,18 +0,0 @@
...
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\Form\Type;
-
-use Sonata\CoreBundle\Form\Type\BaseDoctrineORMSerializationType;
-
-class ApiGalleryItemType extends BaseDoctrineORMSerializationType
```
Comment (jordisala1991) [inline Form/Type/ApiGalleryItemType.php]: And this one

File: PHPCR/BaseGalleryItem.php [hunk 5]
```diff
@@ -1,53 +0,0 @@
...
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\PHPCR;
-
-use Sonata\MediaBundle\Model\GalleryItem;
-
-abstract class BaseGalleryItem extends GalleryItem
```
Comment (jordisala1991) [inline PHPCR/BaseGalleryItem.php]: And this one

File: PHPCR/BaseGalleryItemRepository.php [hunk 6]
```diff
@@ -1,29 +0,0 @@
...
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\PHPCR;
-
-use Doctrine\ODM\PHPCR\DocumentRepository;
-use Doctrine\ODM\PHPCR\Id\RepositoryIdInterface;
-
-class BaseGalleryItemRepository extends DocumentRepository implements RepositoryIdInterface
```
Comment (jordisala1991) [inline PHPCR/BaseGalleryItemRepository.php]: And this class

File: Resources/config/doctrine/BaseGalleryItem.mongodb.xml [hunk 7]
```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jordisala1991) [inline Resources/config/doctrine/BaseGalleryItem.mongodb.xml]: this file should not get removed

File: Resources/config/doctrine/BaseGalleryItem.orm.xml [hunk 8]
```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jordisala1991) [inline Resources/config/doctrine/BaseGalleryItem.orm.xml]: this file should not get removed

File: Resources/config/doctrine/BaseGalleryItem.phpcr.xml [hunk 9]
```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jordisala1991) [inline Resources/config/doctrine/BaseGalleryItem.phpcr.xml]: this file should not get removed

File: Resources/config/doctrine/GalleryItem.orm.xml.skeleton [hunk 10]
```diff
@@ -1,21 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (jordisala1991) [inline Resources/config/doctrine/GalleryItem.orm.xml.skeleton]: this file should not get removed

File: Resources/config/doctrine/GalleryItem.phpcr.xml.skeleton [hunk 11]
```diff
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (jordisala1991) [inline Resources/config/doctrine/GalleryItem.phpcr.xml.skeleton]: this file should not get removed

File: Resources/config/serializer/Model.GalleryItem.xml [hunk 12]
```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jordisala1991) [inline Resources/config/serializer/Model.GalleryItem.xml]: this file should not get removed

File: Tests/Admin/GalleryItemAdminTest.php [hunk 13]
```diff
@@ -1,34 +0,0 @@
...
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\MediaBundle\Tests\Admin;
-
-use PHPUnit\Framework\TestCase;
-use Sonata\MediaBundle\Admin\GalleryItemAdmin;
-
-class GalleryItemAdminTest extends TestCase
```
Comment (jordisala1991) [inline Tests/Admin/GalleryItemAdminTest.php]: this class should not get removed

**Comment by greg0ire** [comment]
I resurrected the files you said.

**Pull request merged**